smart-contracts/test/testCosmosBridge.js

```diff
@@ -63,7 +66,8 @@ describe("Test Cosmos Bridge", function () {
       owner,
       userOne,
       userThree,
-      pauser
+      pauser,
+      networkDescriptor
```

Comment:
Somewhere in this file we should write a test that queries the networkDescriptor from the cosmos bridge and asserts that it was set correctly.